sale_comment_template/views/report_saleorder.xml

```diff
@@ -1,40 +1,24 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-
-    <template id="report_saleorder_document_comments"
-      inherit_id="sale.report_saleorder_document">
-      <xpath expr="//div[hasclass('oe_structure')]" position="before">
-        <template id="sale_css" inherit_id="web.layout">
-            <xpath expr="//head" position="inside">
-                <link rel="stylesheet"
-                    href="/sale_comment_template/static/src/css/sale.css" />
-            </xpath>
-        </template>
-      </xpath>
-      <xpath expr="//table[hasclass('table-sm')]" position="before">
-        <p t-if="doc.note1">
-          <span t-field="doc.note1"/>
-        </p>
-      </xpath>
-
-      <xpath expr="//t[@t-foreach='doc.order_line']" position="inside">
-        <t t-if="line.formatted_note">
-            <tr style="padding:0;">
-                <td colspan="7" style="padding:0;">
-                    <table style="width:100%;border:0;padding:0;">
-                        <caption class="formatted_note">
-                            <span t-field="line.formatted_note"/>
-                        </caption>
-                    </table>
-                </td>
-            </tr>
-        </t>
-      </xpath>
-      <xpath expr="//p[@id='fiscal_position_remark']" position="after">
-        <p t-if="doc.note2">
-          <span t-field="doc.note2"/>
-        </p>
-      </xpath>
+    <template
+        id="report_saleorder_document_comments"
+        inherit_id="sale.report_saleorder_document"
+    >
+        <xpath expr="//table[hasclass('table-sm')]" position="before">
+            <p
```

Comment:
Is this extra `<p>` correct that the we insert inside the HTML raw? Shouldn't it be a `<div>` or similar?